compile.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# Builds all targets and runs all tests
+if [[ ! -z "$CIRCLECI" ]]; then
+  DELETED_PACKAGES="";
+else
+  DELETED_PACKAGES="--deleted_packages $(cat .circleci/deleted_bazel_packages.txt)";
+fi
+
+PROTO_OPTIONS="--proto_compiler //external:proto_compiler --proto_toolchain_for_java //external:proto_java_toolchain"
+
+time bazel build $PROTO_OPTIONS $DELETED_PACKAGES //...
```

Comment:
So now to normally compile (a single target), we need to add the `PROTO_OPTIONS` flag?

Reply:
@oferb yes. is it better to leave `@com_google_protobuf` in `WORKSPACE` or specify it in `README` (I prefer this one)?